pkg/iris/common/models/config.go

```diff
@@ -0,0 +1,7 @@
+package models
+
+type KubernetesConfig struct {
+	ID                    string   `json:"id"`
+	Cluster               string   `json:"cluster"`
+	BlackListedNamespaces []string `json:"blacklistedNamespaces"`
```

Comment:
As mentioned below, the discoveryMode (workload vs namespace) should be a field in the k8s config

Reply:
Added discoveryMode as part of `KubernetesConfig`